ws-nextjs-app/app/api/local/[service]/[pageType]/[id]/[[...optionalParams]]/route.api.ts

```diff
@@ -54,7 +54,6 @@ export async function GET(
     return Response.json(JSON.parse(pageData));
   } catch (error) {
     const { message } = error as FetchError;
-
-    return new Response(message, { status: 500 });
+    return new Response(message, { status: 404 });
```

Comment:
Vaguely remember the discussion about this, but why was this changed from 500 to 404 again?